packages/extension/src/shared/token/storage.ts

```diff
@@ -1,12 +1,11 @@
 import * as yup from "yup"
-
 import { ArrayStorage } from "../storage"
 import { assertSchema } from "../utils/schema"
 import { BaseToken, Token } from "./type"
 import { equalToken, knownAlephiumTokens } from "./utils"
 
 export const tokenStore = new ArrayStorage(knownAlephiumTokens, {
-  namespace: "core:tokens",
+  namespace: "core:tokens-v2",
```

Comment:
Let's just migrate the old version whenever we can, like in this case. A new version would bring additional complexity in the long term because we will need to manage all old versions.